source/patterns/02-organisms/content-header/content-header.twig

```diff
@@ -47,6 +47,19 @@
 
     {%- endwith -%}
 
+    {%- if downloadIconLink is defined -%}
+      {%- with downloadIconLink|merge({html: html}) only -%}
+        {%- set attributes = attributes|default({})|merge({href: attributes.href, class: attributes.class|default([])|merge(['content-header__download_link']) }) -%}
+        <a {{ html.attributes(attributes) }}>
+          <picture>
+            <source srcset="{{asset('images/download-full.svg')}}" type="image/svg+xml" media="(min-width: 45.625em)">
```

Comment:
Will be addressed later by whatever strategy ends up being used to solve https://github.com/libero/libero/issues/151.